api/base/urls.py

```diff
@@ -1,9 +1,12 @@
+from __future__ import print_function
```

Comment:
@pattisdr Something about Travis environment (likely related to there python dist) is causing them to fail without this. [Failing builds](https://travis-ci.org/Johnetordoff/osf.io/jobs/561872814). 

As you can see from the build the DRF's docutils uses a Py3 only print statement, why this doesn't happen locally I don't know.

Reply:
Do we know for a fact that this is happening? What python distribution are they using?

Reply:
If we need to do a TCI-specific fix, it's better if they go in TCI-specific files. Can it be shoved into a relevant test file instead?

Checked the failing build, it used py 2.7.14, trusty, and the same docutils/django/drf that's present on staging servers. It is odd that it would break.